### PR TITLE
pkcs11-tools 2.5.0 (new formula)

### PR DESCRIPTION
**Pull request by aehrisch**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by carlocab — COMMENTED** [review]
Please don't mark items in the PR template as done if you did not do them.

The preferred commit-message style for homebrew/core puts the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

File: Formula/pkcs11-tools.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    # no tests, because all tools require the installation of a PKCS#11 module (shared library).
```
Comment (carlocab) [inline Formula/pkcs11-tools.rb:22]: This doesn't seem like an impediment to testing.

Why can't we install a module for the purposes of testing? Or try to run the tools when we don't have any available to check if we get the behaviour that's expected?

Comment (aehrisch) [reply Formula/pkcs11-tools.rb:22]: The PKCS#11 modules are basically drivers for hardware security modules and only work with specific hardware appliances. I could add a dependency to the softhsm formula which also includes such a module and works without hardware. That dependency would, however, only be necessary for the test cases and not for building or running the pkcs11-tools. If this is acceptable, I would add such tests.

Comment (carlocab) [reply Formula/pkcs11-tools.rb:22]: Do you mean you could make this formula depend on `softhsm`? That's fine, we can always do
```ruby
depends_on "softhsm" => :test
```

**Review by carlocab — APPROVED** [review]
Cleaned your PR up a bit for you. We can merge this when CI is done.

**Pull request closed without merging**